hamocc/mo_param_bgc.F90

```diff
@@ -584,7 +584,7 @@ subroutine ini_bgctimes(nday_in_year)
 
     days_per_year = real(nday_in_year)
 
-    if (nday_in_year /= 365 .and. mnproc==1 .and. lini .eqv. .true.) then
+    if ((nday_in_year /= 365) .and. (mnproc == 1) .and. (lini .eqv. .true.)) then
       lini=.false.
       if (.not. (expcnf == 'single_column' .or. expcnf == 'fuk95' .or. expcnf == 'channel' .or. expcnf == 'noforcing')) then
```

Comment:
Hi @TomasTorsvik , I wonder about this line as well...

Reply:
Yes, in principle it is only the `(lini .eqv. .true. )` that needs the parenthesis, but for consistency I included for the two first statements as well.